KomaMRIFiles/src/Phantom/Phantom.jl

```diff
@@ -5,108 +5,109 @@ Reads a (.phantom) file and creates a Phantom structure from it
 """
 function read_phantom(filename::String)
     fid = HDF5.h5open(filename, "r")
-    phantom_fields = []
-    version = read_attribute(fid, "Version")
     dims = read_attribute(fid, "Dims")
     Ns = read_attribute(fid, "Ns")
+    # Version
+    file_version = VersionNumber(read_attribute(fid, "Version"))
+    program_version = pkgversion(KomaMRIFiles)
+    if file_version.major != program_version.major | file_version.minor != program_version.minor
+        @warn "Version mismatch detected:
+         File version: $file_version
+         KomaMRIFiles version: $program_version
+         This may lead to compatibility issues. Please update the file or the program to the matching version."
```

Comment:
This message could be shorter. And not use new lines. Also, you probably want to check the major version.